modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1855,7 +1872,16 @@ private void positionCell(T cell, double position) {
         }
     }
 
-    private void resizeCellSize(T cell) {
+    /**
+     * Resizes the given cell. For example, if {@link #isVertical()} is set to {@code true}, the cell width will be
+     * maximum between the viewport width and the sum of all the cells {@code prefWidth}. The cell height will be
+     * computed by the cell itself unless {@code fixedCellSizeEnabled} is set to true, then {@link #getFixedCellSize()}
+     * is used.
+     *
```

Comment:
* "will be maximum" -> "will be the maximum"
* "the sum of all the cells" needs apostrophe: cells'
* "is set to true" needs `@code` for `true`
* "For example, if `{@link #isVertical()}` is set to `{@code true}`" if there are only 2 cases (vertical and horizontal), you can drop "for example" and add at the end that for `isVertical` set to `false` the width and height calculations are reversed.